Bottom: 8d055735d616c07a35d1afc72410644c1d567f14
Top:    2312f12dc5b03b1f96c9a4969a8f6af50a4f81f2
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-06-03 23:36:06 +0100

Remove "=" shortcut to the djopt() thingy which keeps eating my design

ARR!! how much work have I lost to this key binding!









---

diff --git a/src/gpcb-menu.res b/src/gpcb-menu.res
index 9be81f5..b695535 100644
--- a/src/gpcb-menu.res
+++ b/src/gpcb-menu.res
@@ -323,13 +323,11 @@ MainMenu =
    {"Rip up all auto-routed tracks" RipUp(All)}
    -
    {"Optimize routed tracks"
-    {"Auto-Optimize" djopt(auto)  a={"Shift-=" "Shift<Key>="}}
     {"Debumpify" djopt(debumpify) }
     {"Unjaggy" djopt(unjaggy) }
     {"Vianudge" djopt(vianudge) }
     {"Viatrim" djopt(viatrim) }
     {"Ortho pull" djopt(orthopull) }
-    {"Simple optimization" djopt(simple)  a={"=" "<Key>="}}
     {"Miter" djopt(miter) }
     {"Puller" a={"Y" "<Key>y"} Puller() }
     {"Global Puller"
